helpdesk_mgmt/models/helpdesk_ticket_team.py

```diff
@@ -0,0 +1,82 @@
+from odoo import api, fields, models
+from odoo.tools.safe_eval import safe_eval
+
+
+class HelpdeskTeam(models.Model):
+
+    _name = "helpdesk.ticket.team"
+    _description = "Helpdesk Ticket Team"
+    _inherit = ["mail.thread", "mail.alias.mixin"]
+
+    name = fields.Char(string="Name", required=True)
+    user_ids = fields.Many2many(comodel_name="res.users", string="Members")
+    active = fields.Boolean(default=True)
+    category_ids = fields.Many2many(
+        comodel_name="helpdesk.ticket.category", string="Category"
+    )
+    company_id = fields.Many2one(
+        comodel_name="res.company",
+        string="Company",
+        default=lambda self: self.env.company,
+    )
+    user_id = fields.Many2one(
+        comodel_name="res.users", string="Team Leader", check_company=True,
+    )
+    alias_id = fields.Many2one(
+        comodel_name="mail.alias",
+        string="Email",
+        ondelete="restrict",
+        required=True,
+        help="The email address associated with \
+                               this channel. New emails received will \
+                               automatically create new tickets assigned \
+                               to the channel.",
+    )
+    color = fields.Integer(string="Color Index", default=0)
+    ticket_ids = fields.One2many(
+        comodel_name="helpdesk.ticket", inverse_name="team_id", string="Tickets",
+    )
+    todo_ticket_ids = fields.One2many(
```

Comment:
@pedrobaeza i think, you better know odoo13 than me. Does it work to have a related field with a domain ?